isisd/isis_route.c

```diff
@@ -48,18 +48,18 @@
 #include "isis_route.h"
 #include "isis_zebra.h"
 
+static struct isis_nexthop *
```

Comment:
I updated the call sites of `nexthoplookup()` in the third commit to check if `nexthoplookup() == NULL` instead of `nexthoplookup() == 0` (I originally did this on the fourth commit, but I agree this change belongs in the third commit).